src/writeData/subscriptions/utils/form.ts

```diff
@@ -82,6 +82,10 @@ export const sanitizeForm = (form: Subscription): Subscription => {
     delete form.brokerUsername
     delete form.brokerPassword
   }
+
+  if (form.dataFormat === 'lineprotocol') {
+    form.timestampPrecision = 'NS'
+  }
```

Comment:
I think we'll need to let users sending LP messages set this property too

Reply:
gotcha, adding now :)